File: php/foo/car.php
<?php

class Car {
    private $honkCount;
    
    public function __construct() {
        $this->honkCount = 0;
    }
    
    public function honk() {
        $this->honkCount++;
        
        return "You honked $this->honkCount times";
    }
    
    public function honkMultipleTimes($num) {
        $this->honkCount += $num;
        
        return "You honked " . $num . " more times for a total of " . $this->honkCount;
    }
}

File: java/example2/src/main/Car.java
public class Car {
    private int honkCount;

    // constructor
    public Car() {
        honkCount = 0;
    }

    public String honk() {
        honkCount++;

        return "You honked " + honkCount + " times";
    }

    public String honkMultipleTimes(int num) {
        honkCount += num;

        return "You honked up to " + num + " more times bringing your total to " + honkCount;
    }
}